pkg/maps/ctmap/types.go

```diff
@@ -346,7 +347,12 @@ func (k *CtKey4Global) Dump(sb *strings.Builder, reverse bool) bool {
 		addrDest = k.DestAddr.String()
 	}
 
-	if k.Flags&TUPLE_F_IN != 0 {
+	if k.Flags&TUPLE_F_SERVICE != 0 {
```

Comment:
Seems this is duplicated for v4/v6. Perhaps we could use a helper?
Also could use Fprintf(sb, ...) vs writeString. Likely these belong in a sep PR??